cmake/Findlibwebsockets.cmake

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
> Can we make the same changes for libunwind as well ?

It would be good if `pkg_check_modules` can be used for libunwind as well.
This PR LGTM. Please commit if CI is good.